modules/social_features/social_book/config/optional/core.entity_view_display.node.book.small_teaser.yml

```diff
@@ -0,0 +1,46 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.small_teaser
+    - field.field.node.book.body
+    - field.field.node.book.field_book_comments
+    - field.field.node.book.field_book_image
+    - field.field.node.book.field_content_visibility
+    - field.field.node.book.field_files
+    - image.style.social_medium
+    - node.type.book
+  module:
+    - image
+    - layout_builder
```

Comment:
are all these dependencies really needed? 🤔

Reply:
I see it was fixed.